src/pages/DulPreview.js

```diff
@@ -24,38 +24,21 @@ class DulPreview extends Component {
 
   async electionReview() {
     const consentId = this.props.match.params.consentId;
-    const consent = await Election.findReviewedElections(consentId, 'TranslateDUL');
-    this.setState({ consentPreview: consent.consent, loading: false });
+    let consent = await Election.findReviewedElections(consentId, 'TranslateDUL');
```

Comment:
This service call looks suspicious, 'TranslatedDUL' parameter seems not being used in ajax.js